testfixtures/logcapture.py

```diff
@@ -60,7 +66,12 @@ def __init__(self, names=None, install=True, level=1, propagate=None,
         self.attributes = attributes
         self.recursive_check = recursive_check
         self.old = defaultdict(dict)
-        self.clear()
```

Comment:
I have requested this more than once now: please put this back.

Reply:
Will do, however I do not agree the result is better, the maintainer has to do extra effort to understand the code.

Reply:
Also lint is on my side as well: "Instance attribute records defined outside __init__"